docs/src/pages/[platform]/getting-started/create-react-app/react.mdx

```diff
@@ -0,0 +1,76 @@
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+import { Example, ExampleCode } from '@/components/Example';
+import { BasicExampleDemo, CompleteExampleDemo } from './examples';
+
+### Run Create-React-App
+
+First, execute the command below in your terminal:
+
+```shell
+npx create-react-app amplify-ui-demo
+```
+
+This creates a basic React app from scratch named `amplify-ui-demo`. For more information, see the [Create-React-App documentation](https://create-react-app.dev).
+
+### Install Amplify UI
+
+If you haven't already, `cd` into the `amplify-ui-demo` folder. Then, install the Amplify UI React package:
+
+<Tabs>
+<TabItem title="npm">
+
+```shell
+npm install aws-amplify @aws-amplify/ui-react
+```
+
+</TabItem>
+<TabItem title="yarn">
+
+```shell
+yarn add aws-amplify @aws-amplify/ui-react
+```
+
+</TabItem>
+</Tabs>
+
+### Basic Demo
+
+Open up the project in your IDE and replace the contents of the `App.js` file with the code below:
+
+<Example>
+  <ExampleCode>
+  
+```tsx file=./examples/BasicExampleCode.tsx
+
+```
+
+  </ExampleCode>
+  <BasicExampleDemo />
+</Example>
+
+When you start the app (`npm/yarn start`), you should see a basic shopping card component. Notice how simple it is to get started with just a few components and style props.
```

Comment:
I think one good rule for docs/technical writing we should try to follow, because it helps create more inclusive/welcoming language, is [avoiding the use of phrases like "simply/simple/easy/etc"](https://www.setcorrect.com/portfolio/work14/). We could probably remove the second sentence completely and/or have a short lead in to the next part of the tutorial (e.g. what we'll build/more advanced usage).